src/util/negateValue.js

```diff
@@ -1,9 +1,18 @@
-import reduceCalc from 'reduce-css-calc'
-
 export default function (value) {
-  try {
-    return reduceCalc(`calc(${value} * -1)`)
-  } catch (e) {
-    return value
+  value = `${value}`
+
+  // Flip sign of numbers
+  if (
+    /^[+-]?(\d+|\d*\.\d+)(e[+-]?\d+)?(%|cm|mm|Q|in|pc|pt|px|em|ex|ch|rem|lh|vw|vh|vmin|vmax)?$/.test(
```

Comment:
Should we maybe simplify this to just `(%|\w+)` so we catch even imaginary units people might be using for other PostCSS plugins or something?